nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core-api/src/main/java/org/apache/nifi/controller/AbstractPort.java

```diff
@@ -497,6 +497,12 @@ public void yield() {
         yieldExpiration.set(Math.max(yieldExpiration.get(), System.currentTimeMillis() + yieldMillis));
     }
 
+    @Override
+    public void yield(final long yieldDuration, final TimeUnit timeUnit) {
```

Comment:
Should the other `yield` method invoke this one?

Reply:
Yes. :)